docs/src/community/iris_xarray.rst

```diff
@@ -107,15 +126,41 @@ However, attribute handling is not so complete: like Iris, it interprets and
 modifies some recognised aspects, and can add some extra attributes not in the
 input.
 
-.. todo:
-    More detail on dates and fill values (@pp-mo suggestion).
-
-Handling of dates and fill values have some special problems here.
-
-Ultimately, nearly everything wanted in a particular desired result file can
-be achieved in Xarray, via provided override mechanisms (`loading keywords`_
+Whereas Iris is primarily designed to handle netCDF data encoded according to
+`CF Conventions <https://cfconventions.org/>`_ , this is not so important to Xarray,
+which therefore may make it harder to correctly manage this type of data.
+While Xarray CF support is not complete, it may improve, and obviously
+:ref:`cfxarray` may be relevant here.
+There is also relevant documentation
+`at this page <https://docs.xarray.dev/en/stable/user-guide/weather-climate.html#weather-and-climate-data>`_.
+
+In some particular aspects, CF data is not loaded well (or at all), and in many cases
+output is not fully CF compliant (as-per `the cf checker <https://cfchecker.ncas.ac.uk/>`_).
+
+* it has it's own interpretation of coordinates, which is different from the CF-based
```

Comment:
It could be made clearer what is being refered to in this list. When we say "it" are we refering to xarray itself, an xarray object or the output of an xarray object?